### PR TITLE
docs(trg1.01): add more detailed information about importance of the readme file

### PR DESCRIPTION
**Pull request by FaGru3n**
## Description

- add more specifc examples
- add link to https://www.makeareadme.com/
- add link to badges https://github.com/badges/shields/blob/master/README.md
- add description of the importance of the `README.md`

fixes https://github.com/eclipse-tractusx/sig-infra/issues/359

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

File: docs/release/trg-1/trg-1-1.md [hunk 1]
```diff
@@ -2,21 +2,40 @@
...
+- Usage Examples and Demonstrations of you product
+- License Information
+- Troubleshooting and FAQs to help users solve common problems
+- Contribution Guidelines to help users contribute to your project
+- Contact Information to help users get in touch with you
+- Professionalism and Credibility for your project
+- Badges to show the status of your project (e.g. build status, code coverage, [etc](https://github.com/badges/shields/blob/master/README.md).)
 
-If installation instructions are too comprehensive for `README.md` move content to `INSTALL.md` file, see [TRG 1.2](trg-1-2.md).
+For our open-source eclipse tractus-x project, the `README.md` often includes guidelines on how others can contribute. This may cover aspects like submitting bug reports, suggesting new features, or providing code contributions. Clear contribution guidelines can help streamline collaboration.
```
Comment (SebastianBezold) [inline docs/release/trg-1/trg-1-1.md:31]: Again, this is covered in our `CONTRIBUTING.md` and should not be duplicated

**Review by SebastianBezold — CHANGES_REQUESTED** [review]

File: docs/release/trg-1/trg-1-1.md [hunk 2]
```diff
@@ -2,21 +2,40 @@
 title: TRG 1.01 - README.md
 ---
 
-| Status | Created      | Post-History |
-|--------|--------------|--------------|
-| Active | 07-Mar-2023  |              |
-| Draft  | 13-Sept-2022 |              |
+| Status | Created      | Post-History                             |
+|--------|--------------|------------------------------------------|
+| Active | 06-Dec-2023  | add detailed information for README file |
```
Comment (hzierer) [inline docs/release/trg-1/trg-1-1.md:7]: two active versions might confuse the audience

File: docs/release/trg-1/trg-1-1.md [hunk 3]
```diff
@@ -2,21 +2,40 @@
...
+| Active | 07-Mar-2023  |                                          |
+| Draft  | 13-Sept-2022 |                                          |
 
 ## Why
 
-A good `README.md` file is the starting point for everyone opening a repository. It should help them find all critical information in an easy way.
+A good `README.md` file is the starting point for everyone opening your repository. It should help them find all critical information in an easy way.
+A well-maintained README reflects positively on the overall professionalism of the project. It shows that the developers care about user experience and want to make it easy for others to understand and use their work.
```
Comment (hzierer) [inline docs/release/trg-1/trg-1-1.md:14]: I would rephrase that a bit: A well-maintained README shows the high level of professionalism...

File: docs/release/trg-1/trg-1-1.md [hunk 4]
```diff
@@ -2,21 +2,40 @@
...
-- If required, additional post installation configuration steps to finish installation
+- Basic documentation of repository and its content to understanding how your repository is structured and what it contains
+- Setup and Installation Instructions to get your component working
+- Configuration and Customization if required for additional post installation steps to finish installation
+- Usage Examples and Demonstrations of you product
+- License Information
+- Troubleshooting and FAQs to help users solve common problems
+- Contribution Guidelines to help users contribute to your project
+- Contact Information to help users get in touch with you
+- Professionalism and Credibility for your project
```
[the remark this answers is not in the document]
Comment (FaGru3n) [reply docs/release/trg-1/trg-1-1.md:28]: like decribed in the end, to show off your professionalism and integrity about your project

> In summary, a good README is a crucial component of any repository, serving as a gateway for users and contributors to understand, use, and potentially contribute to your project. It's a fundamental part of effective project communication and collaboration.

will edit to integrity

File: docs/release/trg-1/trg-1-1.md (hunk 1 above)
Comment (hzierer) [inline docs/release/trg-1/trg-1-1.md:31]: Duplicate to line 26 - please merge these two.

File: docs/release/trg-1/trg-1-1.md [hunk 5]
```diff
@@ -2,21 +2,40 @@
...
+- Badges to show the status of your project (e.g. build status, code coverage, [etc](https://github.com/badges/shields/blob/master/README.md).)
 
-If installation instructions are too comprehensive for `README.md` move content to `INSTALL.md` file, see [TRG 1.2](trg-1-2.md).
+For our open-source eclipse tractus-x project, the `README.md` often includes guidelines on how others can contribute. This may cover aspects like submitting bug reports, suggesting new features, or providing code contributions. Clear contribution guidelines can help streamline collaboration.
+
+Common questions that are answered in the `README.md` file include:
+
+If installation instructions for your product are too comprehensive for `README.md` move content to `INSTALL.md` file, see [TRG 1.02](trg-1-2.md).
```
Comment (hzierer) [inline docs/release/trg-1/trg-1-1.md:35]: proposal:
comprehensive -> extensive or large
...for README.md please move content...

**Review by hzierer — CHANGES_REQUESTED** [review]
see my comments and proposals inline

File: docs/release/trg-1/trg-1-1.md [hunk 6]
```diff
@@ -2,21 +2,40 @@
...
 ## Description
 
 Each repository **must** contain a `README.md` file which **shall** cover following topics:
 
-- Basic description of repository and its content
-- Installation instructions to get component working
-- If required, additional post installation configuration steps to finish installation
+- Basic documentation of repository and its content to understanding how your repository is structured and what it contains
+- Setup and Installation Instructions to get your component working
+- Configuration and Customization if required for additional post installation steps to finish installation
```
[the remark this answers is not in the document]
Comment (FaGru3n) [reply docs/release/trg-1/trg-1-1.md]: [f606a4c](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/530/commits/f606a4c21c791e0a6615e7c1fab7f0feec2dd855)

File: docs/release/trg-1/trg-1-1.md [hunk 7]
```diff
@@ -2,21 +2,40 @@
...
+
+Common questions that are answered in the `README.md` file include:
+
+If installation instructions for your product are too comprehensive for `README.md` move content to `INSTALL.md` file, see [TRG 1.02](trg-1-2.md).
+If your project is too complex to be described in a single `README.md` file, consider creating a `docs` folder and adding a `README.md` file there. This will help you keep your main repository directory clean and tidy.
+If your project has multiple component repositories, link the other repositories in your leading repository in your `README.md` against to it.
+
+In summary, a good README is a crucial component of any repository, serving as a gateway for users and contributors to understand, use, and potentially contribute to your project. It's a fundamental part of effective project communication and collaboration.
+
+Further [example ideas](https://www.makeareadme.com/) how to create your `README.md` file.
```
[the remark this answers is not in the document]
Comment (FaGru3n) [reply docs/release/trg-1/trg-1-1.md]: [4090379](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/530/commits/409037992f6ff3f7f93eb27e04e78edef5840b00)

File: docs/release/trg-1/trg-1-1.md [hunk 8]
```diff
@@ -2,21 +2,40 @@
...
+- Badges to show the status of your project (e.g. build status, code coverage, [etc](https://github.com/badges/shields/blob/master/README.md).)
 
-If installation instructions are too comprehensive for `README.md` move content to `INSTALL.md` file, see [TRG 1.2](trg-1-2.md).
+For our open-source eclipse tractus-x project, the `README.md` often includes guidelines on how others can contribute. This may cover aspects like submitting bug reports, suggesting new features, or providing code contributions. Clear contribution guidelines can help streamline collaboration.
+
+Common questions that are answered in the `README.md` file include:
+
+If installation instructions for your product are too comprehensive for `README.md` move content to `INSTALL.md` file, see [TRG 1.02](trg-1-2.md).
+If your project is too complex to be described in a single `README.md` file, consider creating a `docs` folder and adding a `README.md` file there. This will help you keep your main repository directory clean and tidy.
+If your project has multiple component repositories, link the other repositories in your leading repository in your `README.md` against to it.
```
[the remark this answers is not in the document]
Comment (FaGru3n) [reply docs/release/trg-1/trg-1-1.md]: [4090379](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/530/commits/409037992f6ff3f7f93eb27e04e78edef5840b00)

File: docs/release/trg-1/trg-1-1.md (hunk 5 above)
Comment (FaGru3n) [reply docs/release/trg-1/trg-1-1.md]: [4090379](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/530/commits/409037992f6ff3f7f93eb27e04e78edef5840b00)

File: docs/release/trg-1/trg-1-1.md (hunk 1 above)
Comment (FaGru3n) [reply docs/release/trg-1/trg-1-1.md]: [4090379](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/530/commits/409037992f6ff3f7f93eb27e04e78edef5840b00)

File: docs/release/trg-1/trg-1-1.md (hunk 3 above)
Comment (FaGru3n) [reply docs/release/trg-1/trg-1-1.md]: [4090379](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/530/commits/409037992f6ff3f7f93eb27e04e78edef5840b00)

**Review by hzierer — APPROVED** [review]
LGTM!

**Review by SebastianBezold — APPROVED** [review]

File: docs/release/trg-1/trg-1-1.md [hunk 9]
```diff
@@ -2,21 +2,37 @@
...
+- Troubleshooting and FAQs to help users solve common problems
+- Reference your Contribution guideline in the `CONTRIBUTING.md` file
+- Contact information to help users getting in touch with you
+- Badges to show the status of your project (e.g. build status, code coverage, [etc](https://github.com/badges/shields/blob/master/README.md).)
 
-If installation instructions are too comprehensive for `README.md` move content to `INSTALL.md` file, see [TRG 1.2](trg-1-2.md).
+Best practices:
+
+- If installation instructions for your product are too complex being part of `README.md` move its content to a separate `INSTALL.md` file, see [TRG 1.02](trg-1-2.md).
+- If your project is too complex to be described in a single `README.md` file, consider creating a `docs` folder and link your `README.md` to it.
```
Comment (tomaszbarwicki) [inline docs/release/trg-1/trg-1-1.md:33]: Not sure if that statement doesn't bring a little confusion which might be percived as 'docs' being optional, however 'docs' is mandatory anyway despite of having complex or not README. 
Would maybe reword it slightly like "it may be beneficial to relocate specific sections to a dedicated docs/ folder"

Comment (FaGru3n) [reply docs/release/trg-1/trg-1-1.md]: [cde0086](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/530/commits/cde008647f7011db8e47929d62c096456082e57a)

**Review by almadigabor — APPROVED** [review]
LGTM

**Review by hzierer — APPROVED** [review]
👍

**Review by SebastianBezold — APPROVED** [review]
